### ISSUE TITLE
Update Compatibility Comment in README.MD

### ISSUE THREAD

**Issue by a093130** [esa/pagmo_plugins_nonfree#29]
Just a nit, but very confusing.  The SNOPT 7.7 tickets # 6 and #7 have been closed since Jan 2019.  yet the README.md file still says, 
 
>WARNING: **plugin compatible with worhp 1.14** and **snopt < 7.7** (with the snopt7_c interface built by the github project snopt->interface, commit 76b166ecdf5c55a3289ce0f849d8d3d101954a22.). It will not work with worhp 1.13 or any other version, nor snopt 7.7 >(headers were changed upstream).

**Comment by darioizzo** [comment]
Yes, you are right. We use 7.2 regularly... the 7.7 has been built and tested but no user should far confirmed its working as expected hence I guess we have not advertised it "much" .... PR welcome especially with a further independent test.

**Comment by darioizzo** [comment]
closing this
